oops/oops.go

```diff
@@ -38,6 +38,47 @@ func (e *oopsError) Error() string {
 	return buffer.String()
 }
 
+// MainStackToString will write the frames of the main goroutine to a string.
+// This will return an empty string if the error is not an oopsError or it cannot retrieve the frame from the error.
+func MainStackToString(err error) string {
```

Comment:
maybe i'm flipping back. i wonder if it's more useful to always print only the first stack and allow others to pull the further down stacks if needed?

(i.e. make the `Error()` function on the oops error return only the first stack by default

Reply:
I think if we want a trimmed version as the default, it should include at least the first N rather than just one